disnake/components.py

```diff
@@ -50,7 +50,6 @@
     "Button",
     "SelectMenu",
     "SelectOption",
-    "Modal",
```

Comment:
Sadly, I can't do this suggestion above, but this import is now unused. 
![image](https://user-images.githubusercontent.com/67214928/151225389-13e2818a-fd68-4f98-8f66-b2188c9de1e0.png)